kube-enforcer/templates/cluster-role.yaml

```diff
@@ -141,6 +144,23 @@ rules:
     verbs:
       - create
       - delete
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - networkpolicies
+      - ingresses
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - policy
+    resources:
+      - podsecuritypolicies
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
Just a question @KoppulaRajender @semyonmor the list below:

```
  - vulnerabilityreports
      - configauditreports
      - clusterconfigauditreports
      - ciskubebenchreports
      ```

Looks to be missing a number of resources as per errors I am seeing in the controller and based on cross-checking with the chart over in the starboard-operator repo.

I also noticve that this chart does not install all the CRDs needed for the operator to start up correctly, I ended up bringing them in out-of-band which is how we manage CRDs but I did not see anything in the docs teliling me I needed to do this so it was a gotcha for us.

Any plans to perhaps install starboard-operator as a nested chart?

Reply:
Hi @stefansedich,
Currently not all features for Starboard in integrated with KubeEnforcer, so we are only creating the CRDs which are integrated with KubeEnforcer and the rest are disabled in Starboard from env variables. And that's why we have these differences. In the future, we will integrate it fully. 
But currently, you shouldn't get errors in starboard as long as these options are disabled `OPERATOR_CLUSTER_COMPLIANCE_ENABLED`,  `OPERATOR_VULNERABILITY_SCANNER_ENABLED`  and `OPERATOR_CIS_KUBERNETES_BENCHMARK_ENABLED`